src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,8 @@
+import style from './Footer.module.scss';
+
+export const Footer = () => (
+  <footer className={style.footer}>
+    <p>©</p>
+    <p>GitHub</p>
```

Comment:
github? :D Maybe KamcaLabs ? 
```suggestion
    <p>GitHub</p>
```